server/devtool/webeditor/src/main/resources/application-local.yml

```diff
@@ -3,23 +3,23 @@ vorto:
   repository:
     base:
       path: http://vorto.eclipse.org
-      
+
 http:
   proxyHost:
   proxyPort: 8080
 
 server:
   port: 9080
   contextPath: /editor
-  
+
 github:
   client:
-    clientId: ${github_clientid}
-    clientSecret: ${github_secret}
+    clientId: b75406b8428b88732920
```

Comment:
Thanks for pointing that out. 
Removed these credentials and created a new commit